cloudbuild.yaml

```diff
@@ -110,17 +112,45 @@ steps:
     apk add --update --no-cache jq
     jq -f backend-service-template.jq neg-status.json | \
     sed "s/$\[PROJECT_ID\]/${PROJECT_ID}/g" | \
-    sed "s/$\[BACKEND_SERVICE_HOST_NAME\]/${_BACKEND_SERVICE_HOST_NAME}/g" > backend-service-hydrated.json
+    sed "s/$\[BACKEND_RESOURCE_NAME\]/web-app-backend-api-service/g" | \
```

Comment:
Don't want to add that to this PR, it's big enough already. I do want to do couple refactor on the script. Adding cluster name will be part of it. Another part will be extracting the part related to add a backend service, mainly setup backend service and url mapping, so that it's easier to add more. But again, a separate PR/issue.